### PR TITLE
fix(test_default): Switch oracle db version for gemini

### PR DESCRIPTION
**Pull request by aleksbykov**
Change oracle db version for gemini tests

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by juliayakovlev — APPROVED** [review]

**Review by fruch — CHANGES_REQUESTED** [review]
@aleksbykov unittest are failing cause of this change, please fix:
```
05:01:23  FAILED unit_tests/test_config.py::ConfigurationTests::test_16_default_oracle_scylla_version_eu_west_1
05:01:23  FAILED unit_tests/test_scylla_yaml_builders.py::IntegrationTests::test_integration_node_2__jenkins_slave_workspace_can_scylla_cluster_tests_PR_5411_unit_tests_test_data_test_scylla_yaml_builders_cdc_replication_longevity_yaml
```

File: defaults/test_default.yaml [hunk 1]
```diff
@@ -114,7 +114,7 @@ authenticator_password: ''
 # gemini defaults
 n_test_oracle_db_nodes: 1
 gemini_seed: 0
-oracle_scylla_version: '4.4.7'
+oracle_scylla_version: '4.6.9'
```
Comment (fgelcer) [inline defaults/test_default.yaml:117]: why not 5.0?

**Review by fgelcer — APPROVED** [review]
LGTM

File: defaults/test_default.yaml (hunk 1 above)
Comment (fruch) [reply defaults/test_default.yaml:117]: seems that @aleksbykov want to backport this to 2022.1

I would recommend switching to oracle 5.0 on master/2022.2/5.1, while doing a separate PRs for 5.1/2022.1

Comment (fgelcer) [reply defaults/test_default.yaml:117]: agree

Comment (aleksbykov) [reply defaults/test_default.yaml:117]: As we agreed long time ago, we use oldest supporting scylla version

**Comment by aleksbykov** [comment]
> @aleksbykov unittest are failing cause of this change, please fix:
> 
> ```
> 05:01:23  FAILED unit_tests/test_config.py::ConfigurationTests::test_16_default_oracle_scylla_version_eu_west_1
> 05:01:23  FAILED unit_tests/test_scylla_yaml_builders.py::IntegrationTests::test_integration_node_2__jenkins_slave_workspace_can_scylla_cluster_tests_PR_5411_unit_tests_test_data_test_scylla_yaml_builders_cdc_replication_longevity_yaml
> ```

Fixed

**Review by fruch — APPROVED** [review]

**Pull request merged**